datatypes/src/primitives/feature_data.rs

```diff
@@ -1227,6 +1227,8 @@ impl TryFrom<&FeatureDataValue> for i64 {
         Ok(match value {
             FeatureDataValue::Int(v) => *v,
             FeatureDataValue::NullableInt(v) if v.is_some() => v.unwrap(),
+            FeatureDataValue::DateTime(v) => v.inner(),
+            FeatureDataValue::NullableDateTime(v) if v.is_some() => v.unwrap().inner(),
```

Comment:
While I'm seeing this: Can't we just say
```suggestion
            FeatureDataValue::DateTime(v) => v.inner(),
            FeatureDataValue::NullableDateTime(Some(v)) => v.inner(),
```
?

The same holds for the match above.

Reply:
That worked and it also allowed the match arms to be combined. I changed the other places accordingly: 2013135